### ISSUE TITLE
No output, breaks the page.

### ISSUE THREAD

**Issue by jpmurray** [sekati/codeigniter-asset-helper#2]
Hi,

I just installed via sparks. Checking the complete log of code igniter, everything seems fine (config and helper are loaded).

![Untitled-2](https://f.cloud.github.com/assets/1550428/132245/587d3afe-707e-11e2-92ca-579cf88a43ab.jpg)

At this point, loading a page prints everything correctly to the screen (and HTML source).

Then, trying to implement the functionnality of the asset helper, with any function, but for this example, trying to print <?=css_url()?> won't work, and looking at the source code, it just breaks where the echoing is supposed to be done.

![Untitled-2](https://f.cloud.github.com/assets/1550428/132241/2178979c-707e-11e2-96f1-d53e10531c63.jpg)

I've tried to try and debug as much as I could, but I'm clueless here as to the why and what to do to correct this. Any idea?

**Comment by sekati** [comment]
This looks like a possible PHP issue rather than an issue with the spark - can you confirm you have short tags enabled (see links below) or use the non-short tag syntax. e.g: <?php echo( css_url() . 'my.css' ); ?>

http://ellislab.com/codeigniter/user-guide/general/alternative_php.html
http://www.php.net/manual/en/ini.core.php#ini.short-open-tag

**Comment by jpmurray** [comment]
Yep, shortags do works (just tried an <?="hello world"?> succesfully. I've also tried <?php echo( css_url() . 'my.css' ); ?> without success.

Now it's my first time working with code igniter, so I've enabled every error messages possibles in the log, and unless I'm mistaken, any php error should come up in the log, no? I've just checked also, just to be sure, php.ini error reporting should at least throw something in case of an error and still nothing.

EDIT: I decided to explore a little, and I went to comment everything in the css_url function. Modifying it so it just returns an "hello world" still won't do. I'm really puzzeled.
